examples/api-tests/src/find-replace.spec.js

```diff
@@ -59,18 +58,28 @@ describe('Find and Replace', function () {
         });
     }
 
+    const delay = 500;
+    function pause(t = delay) {
+        console.debug(`pause test for: ${t} ms`);
+        return new Promise(resolve => setTimeout(resolve, t));
+    }
+
+
```

Comment:
```suggestion
    function pause(ms = 500) {
        console.debug(`Pause test for: ${ms} ms`);
        return new Promise(resolve => setTimeout(resolve, ms));
    }

```

Reply:
Paul, I appreciate you using the GitHub "suggestion" mechanism. However I think it will add commits on top? If that's ok with you, I would rather do the changes in the corresponding commit?

Reply:
I have done so, adding a note in the commit message to credit the fact that I have included a suggested change from you.